keras_cv/layers/preprocessing/base_image_augmentation_layer.py

```diff
@@ -293,7 +293,14 @@ def _augment(self, inputs):
         return result
 
     def _batch_augment(self, inputs):
-        return self._map_fn(self._augment, inputs)
+        if BOUNDING_BOXES in inputs:
```

Comment:
if users provide us with already padded bounding boxes, we should return padded bounding boxes.  As such, we need to use the `_format_inputs()` pattern here, and store whether or not the bounding box inputs were padded or not.

Reply:
done!